freeplane/src/main/java/org/freeplane/features/link/mindmapmode/MLinkController.java

```diff
@@ -1136,4 +1141,28 @@ public void act() {
 			modeController.execute(actor, map);
 		}
 	}
+
+	public void addConnectorWithNodeDependantStyle(final NodeModel target, NodeModel node) {
```

Comment:
The order of arguments is different from a similar method `addConnector(node, target)` vs `(target, node)` - is this intentional?

Reply:
signature changed